src/utils/assetManagers/assetManagers.test.js

```diff
@@ -271,4 +271,27 @@ describe('utils/assetManagers', () => {
       })
     })
   })
+
+  describe('retrieveEODBooks', () => {
```

Comment:
The `mockImplementation` should be on `network.retrieveData`. Since that is the function that is used by `retrieveEODBooks()`